src/cli/ionic-config.ts

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs-extra';
```

Comment:
I think `fs-extra` is a dependency that won't be available in the build. You might be able to use
```suggestion
import { promises as fs } from 'fs';
```

Reply:
Well I'm glad I wrote those tests! I switched this out, worked great.